actors/builtin/miner/miner_test.go

```diff
@@ -345,7 +346,8 @@ func TestCommitments(t *testing.T) {
 		rt.SetEpoch(periodOffset + miner.WPoStChallengeWindow)
 
 		// Commit a sector to upgrade
-		oldSector := actor.commitAndProveSectors(rt, 1, 181, nil)[0]
+		// Use the max sector number to make sure everything works.
+		oldSector := actor.commitAndProveSector(rt, abi.MaxSectorNumber, 181, nil)
```

Comment:
@anorth would you rather I create new tests for things like this?

Reply:
It's a pragmatic balance, this case is fine. What's missing, though, is a test that `abi.MaxSectorNumber+1` is rejected.